tuts/start-a-private-network/v2.0.0-alpha.5/customchain.md

```diff
@@ -1,99 +1,10 @@
 ---
 slug: customchain
 lang: en
-title: Creating Your Private Network
+title: Launch Your Private Network
 ---
-Now that each participant has their own keys generated, you're ready to start your own custom chain. In this section it is no longer required to use a single physical machine or a single binary.
```

Comment:
Note to reviewers. This section is not removed entirely. It is split into its own file `customspec.md`.